docs/sketch-specification.md

```diff
@@ -68,14 +68,12 @@ the `data` folder, so any non-code files outside the `data` folder are stripped.
 
 #### `sketch.json`
 
-Arduino CLI and Arduino Web Editor use a file named sketch.json, located in the sketch root folder, to store sketch
-metadata.
+Arduino Web Editor use a file named `sketch.json`, located in the sketch root folder, to store sketch metadata. This
+file is not used by the Arduino CLI or the Arduino IDE, if you're not an Arduino Web Editor user you can safely ignore
+it.
```

Comment:
```suggestion
Arduino Web Editor uses a file named `sketch.json`, located in the sketch root folder, to store sketch metadata. This
file is not used by Arduino CLI or Arduino IDE, if you're not an Arduino Web Editor user you can safely ignore the file.
```

- Grammar fix
- Remove some occurrences of "the" (they seemed superfluous to me)
- Make it clear what "it" was referring to (it seemed a little ambiguous to me)